tests/unit/ViolationUtilsTest.ts

```diff
@@ -60,6 +61,16 @@ describe('getViolationsOnyxData', () => {
         expect(result.value).toEqual(expect.arrayContaining(transactionViolations));
     });
 
+    it('should not add violation when the transaction is partial', () => {
+        const partialTransaction = {...transaction, amount: 0, merchant: CONST.TRANSACTION.PARTIAL_TRANSACTION_MERCHANT};
+        transactionViolations = [
+            {name: 'duplicatedTransaction', type: 'violation'},
+            {name: 'receiptRequired', type: 'violation'},
```

Comment:
Let's use consts for this. I think we need to add one for `duplicatedTransaction`
```suggestion
            {name: CONST.VIOLATIONS.RECEIPT_REQUIRED, type: 'violation'},
```

Reply:
I have updated the test to not have an existing violation for partial transaction anymore